itests/standalone/smoke/src/test/java/org/wildfly/camel/test/spring/subE/ws/WebServiceEndpointImpl.java

```diff
@@ -33,7 +33,7 @@
     @Resource(mappedName = "java:jboss/camel/context/jndi-delayed-binding-spring-context")
     private CamelContext camelctxB;
 
-    @Resource(name = "java:jboss/camel/context/jndi-delayed-binding-spring-context")
+    @Resource(lookup = "java:jboss/camel/context/jndi-delayed-binding-spring-context")
```

Comment:
For this test, either we keep the 'name' attribute or remove this line altogether. We already have a `@Resource` injection using 'lookup' previously defined.